assets/styles/tools/_mixins.scss

```
@@ -193,3 +193,34 @@
     display: $display $important;
     visibility: visible  $important;
 }
+
+// Aspect-ratio polyfill
+//
+// The element will be accessible from screen readers and visible in the document flow.
+//
+// @param  {Number} $ratio      [19/6]  - The ratio of the element.
+// @param  {Number} $width      [100%]  - The width of element.
+// @param  {Boolean} $children  [false] - Whether the element contains children.
```

Fix alignment and improve description of parameters.

```suggestion
// @param  {Number}  $ratio     [19/6]  - The ratio of the element.
// @param  {Number}  $width     [100%]  - The fallback width of element.
// @param  {Boolean} $children  [false] - Whether the element contains children for the fallback properties.
```